dev/deps/spark-deps-hadoop-3.2-hive-2.3

```diff
@@ -118,6 +118,7 @@ jackson-module-scala_2.12/2.10.5//jackson-module-scala_2.12-2.10.5.jar
 jakarta.activation-api/1.2.1//jakarta.activation-api-1.2.1.jar
 jakarta.annotation-api/1.3.5//jakarta.annotation-api-1.3.5.jar
 jakarta.inject/2.6.1//jakarta.inject-2.6.1.jar
+jakarta.servlet-api/4.0.3//jakarta.servlet-api-4.0.3.jar
```

Comment:
I just realised that we still add `javax.servlet-api/3.1.0//javax.servlet-api-3.1.0.jar` below in Hadoop 3. Do you know why this is not excluded? Looks like it can conflict.